Automattic/wp-calypso
# Blog onboarding: Remove stepper navigation hook

Fixing a typescript error by properly clearing out the stepper navigation hook which isn't required for this flow. 

The typescript error was where I was attempting to use the current step string as the navigate function which comes in param position 2, not 1.

Fixes https://github.com/Automattic/wp-calypso/issues/78086

Related to https://github.com/Automattic/wp-calypso/pull/78078

## Proposed Changes

* Do a better job at stubbing out `useStepNavigation`.

## Testing Instructions

* On a new user or a user without any sites
* http://calypso.localhost:3000/setup/blog
* Clicking action buttons should take you to the appropriate flow

![Screenshot 2023-06-13 at 14-15-00 Create your blog — WordPress com](https://github.com/Automattic/wp-calypso/assets/811776/14288649-8ffd-4c9c-a07b-609caaaa2843)
